attributes/default.rb

```diff
@@ -221,6 +221,11 @@
 # extra_packages to install
 default['datadog']['extra_packages'] = {}
 
+# extra config options
+# If an agent is released with a new config option which is not yet supported by this cookbook
+# you can use this attribute to set it. Will be ignored if falsy.
```

Comment:
Makes sense. I didn't think of `false`, only of `''`, which is useless since ini requires key: value.